rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/does_each_zone_have_only_one_terminal.py

```diff
@@ -0,0 +1,25 @@
+from rct229.utils.jsonpath_utils import find_exactly_one_with_field_value
+
+
+def does_each_zone_have_only_one_terminal(rmi_b, zone_id_list):
+    """Returns TRUE if each zone input to this function only has one terminal unit. It returns FALSE if any zone has more than one terminal unit.
+    ----------
+    rmi_b : json
+        RMD at RuleSetModelInstance level
+    zone_id_list : list
+        List of zones to evaluate.
+    Returns
+    -------
+    bool
+        True: each zone input to this function only has one terminal unit or no terminal
+        False: any zone has more than one terminal
+    """
+    does_each_zone_have_only_one_terminal_flag = True
+    for zone_id in zone_id_list:
+        zone = find_exactly_one_with_field_value(
+            "$.buildings[*].building_segments[*].zones", "id", zone_id, rmi_b
+        )
+        if zone.get("terminals") is not None and len(zone["terminals"]) > 1:
```

Comment:
Could this logic also determine ```no terminal units with the zone``` case?

Reply:
It is possible. I posted this issue [here ](https://github.com/pnnl/ruleset-checking-tool/issues/627)for the `no terminal` situation